README.md

```diff
@@ -15,9 +15,10 @@ The task will:
 
 There're some steps required to successfully deploy the automation. 
 
-1. Setup AWS Secrets Manager `secret` with Zanshin API Key
-2. Choose VPC subnets that can reach the Internet
-3. Privileges to deploy CloudFormation Stacks on AWS Management Account
+1. Generate an Zanshin API Key for a user with admin privileges
+2. Setup AWS Secrets Manager `secret` with Zanshin API Key
+3. Choose a VPC private/public subnet that can reach the Internet
+4. Privileges to deploy CloudFormation Stacks on AWS Management Account
```

Comment:
Missing a verb. "Give privilege"?

Reply:
@edpin, in this case, having the privileges are pre-requisites to deploy the cloud formation. It inst about giving additional privileges.